python/ray/serve/scripts.py

```diff
@@ -100,6 +107,35 @@ def process_dict_for_yaml_dump(data):
     return data
 
 
+def print_status_for_app(name: str, application: ApplicationDetails):
+    status = json.loads(
+        ServeStatusSchema(
```

Comment:
Could you make a method in the `ApplicationDetails` class that returns the `ServeStatusSchema` of its contents? You could call that function here, and the we can reuse it throughout the codebase.

Reply:
Added, thanks for the suggestion!